docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -31,13 +28,34 @@ Run the following command:
   ```
 
 #### **Kyma Dashboard**
-1. Navigate to `Cluster Details`.
-2. Open the Istio module's configuration.
-3. Click **Edit**.
-4. Add the number of trusted proxies and click **Update**.
-   ![Add the numTrustedProxies](./assets/01-00-num-trusted-proxies-ui.svg)
+1. Navigate to Cluster Details and select Modify Modules.
+2. Choose the Istio module and select Edit.
+3. In the General section, add the number of trusted proxies.
+4. Select Save.
 <!-- tabs:end -->
 
+### Configure the externalTrafficPolicy in the Istio Custom Resource (GCP and Azure only)
+
+To make XFF header contain client's IP on GCP and Azure, it's required to set gatewayExternalTrafficPolicy to `Local`
+
+Hint: While using GCP or Azure you can find your load balancer's IP address in the ingress-gateway Service's status.loadBalancer.ingress field.
+
+Add **gatewayExternalTrafficPolicy** to the Istio custom resource:
+
+<!-- tabs:start -->
+#### **kubectl**
+Run the following command:
+
+  ```bash
+  kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"config":{"gatewayExternalTrafficPolicy": "Local"}}}'
+  ```
+
+#### **Kyma Dashboard**
+1. Navigate to Cluster Details and select Modify Modules.
+2. Choose the Istio module and select Edit.
+3. In the General section, select `Local` in gatewayExternalTrafficPolicy field.
```

Comment:
```suggestion
3. In the `General` section, set the external traffic policy for Istio Ingress Gateway to `Local`.
```

Reply:
The ui field name is `gatewayExternalTrafficPolicy`, as so it might be confusing for the user if we don't use the exact name here

Reply:
But it's not visible in Busola, right? I'd use the same name here as in Busola. So, "Gateway external traffic policy" if we want to stick with it

Reply:
Right, here it makes sense as this an UI tutorial, my bad